Shopify/shopify-cli
# pass extension title and name

### WHY are these changes introduced?

Part of https://github.com/Shopify/shopify-cli-extensions/issues/150

### WHAT is this pull request doing?

* Parse title from `.env` and parse `name` from config yml file.

### How to test your changes?

* within the shopify-cli repo, run rake extensions:install
* give yourself the required beta flag: shopify config feature extension_server_beta --enable
* cd into an extension project
* [CHECKOUT_UI_EXTENSIONS only] run echo "EXTENSION_RESOURCE_URL=/cart/1:1" >> .env in the extension project (this bypasses the call to the AdminAPI to fetch product data to build the resource_url, as this call is intermittently failing for some users)
* add a name to your  config file: `echo "name: 'title'" >> shopifile.yml` 
* run shopify extension serve

You should see

<img width="1094" alt="Screen Shot 2021-11-09 at 8 55 08 AM" src="https://user-images.githubusercontent.com/989784/140947474-539bde31-fa47-48d7-874c-63181776bb5c.png">

### Update checklist

~- [ ] I've added a CHANGELOG entry for this PR (if the change is public-facing)~ not external facing yet
